src/jquery.valueview.valueview.js

```diff
@@ -811,6 +811,10 @@ $.widget( 'valueview.valueview', PARENT, {
 						// Another reason for this could be something like "a", "ab", "a", where the
 						// first response comes back and the following two can be ignored.
 						self.__lastUpdateValue = undefined;
+					} else {
+						// the value has since been updated, so should be re-parsed
+						// and this result be rejected and ignored.
+						deferred.reject();
```

Comment:
i can combine these specifically.

in general, i think a bit of refactoring + more tests could be done, probably as follow up.